.gitignore

```diff
@@ -1,5 +1,5 @@
 commit_hash.txt
-prerelease.txt
+# prerelease.txt
```

Comment:
What is prerelease.txt?

Reply:
The presence of `prerelease.txt` signals a release version of solidity. Necessary to be compatible with `pragma solidity 0.6.10`